torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16196,10 +16219,15 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.skip('Skipped!'), 'TestCommon', 'test_variant_consistency_eager'),
                DecorateInfo(unittest.skip('Skipped!'), 'TestJit', 'test_variant_consistency_jit'),
                DecorateInfo(unittest.skip('Skipped!'), 'TestMathBits', 'test_neg_view'),
-               # NotImplementedError: Could not run 'aten::tril_indices' with arguments from the 'Meta' backend.
-               DecorateInfo(unittest.expectedFailure, 'TestFakeTensorNonErroring', 'test_fake_autocast'),
-               DecorateInfo(unittest.expectedFailure, 'TestFakeTensorNonErroring', 'test_fake'),
-               DecorateInfo(unittest.expectedFailure, 'TestProxyTensorOpInfo', 'test_make_fx_fake_exhaustive'),
+               # test_dtypes will find this works for float dtypes, but it doesn't really
+               # make sense to use this with floats
+               DecorateInfo(unittest.skip('Skipped!'), 'TestCommon', 'test_dtypes'),
```

Comment:
Then should this function support a non-integral dtype? Perhaps we should add a check on the ATen implementation... WDYT @ngimel?

Reply:
The implementations do *mostly* work for float indices, and maybe there is a use case for having it and I am just not seeing it.

To be honest, the reason I restricted to ints there is that there are some test failures for large sizes that only happen for float16/bfloat16 and it felt like it didn't really make sense to support that.

Reply:
Yeah agree with @lezcano, this function should naturally support only integral dtypes (and only reasonably wide integral dtypes at that, not uint8, which is just asking for errors)

Reply:
OK, cool, I will make those changes then.